app/cdc_uart_bridge.c

```diff
@@ -58,7 +57,8 @@ static void rx_task(void *p_arg)
 				;
 		}
 
-		r = uart_tx(rx_queue_item.data, rx_queue_item.len, portMAX_DELAY, true);
+		usb_cdc_tx(rx_queue_item.data, rx_queue_item.len);
```

Comment:
so.... mirror all data back?

Reply:
As I wrote before it’s a bit rough patch right now. Need to clean up stuff like this and some #if 0